docs/openapi/api-internal-pn-bff-tos-privacy.yaml

```diff
@@ -79,4 +79,57 @@ paths:
           content:
             application/problem+json:
               schema:
-                $ref: './common-refs.yaml#/components/schemas/Problem'
\ No newline at end of file
+                $ref: './common-refs.yaml#/components/schemas/Problem'
+
+  '/bff/v1/pg/tos-privacy':
+    parameters:
+     - $ref: './common-refs.yaml#/components/parameters/uidAuthFleet'             # NO EXTERNAL
+     - $ref: './common-refs.yaml#/components/parameters/cxTypeAuthFleet'          # NO EXTERNAL
+     - $ref: './tos-privacy-schemas/tos-privacy.yaml#/components/parameters/BffQueryTosPrivacyType'
+    get:
+      summary: Get single consent by type
+      description: Returns single consent type for the recipient. Return a Consent with accepted false if consent type is not found.
+      operationId: getPgTosPrivacyV1
+      tags:
+        - UserConsents
+      #      security:                                                                # ONLY EXTERNAL
+      #        - bearerAuth: [ ]                                                      # ONLY EXTERNAL
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: './tos-privacy-schemas/tos-privacy.yaml#/components/schemas/BffTosPrivacyConsent'
+        '400':
+          description: Invalid input
+          content:
+            application/problem+json:
+              schema:
+                $ref: './common-refs.yaml#/components/schemas/Problem'
+    put:
+      summary: Accept a single consent type
+      description: Accept single consent type for the recipient
```

Comment:
Use same description of /bff/v2/tos-privacy specifying that it is for pg

Reply:
done

Reply:
you haven't specified that it is for pg

Reply:
done